ant/self-sign.properties

```diff
@@ -14,6 +14,7 @@ jks.alias=${build.socket.name}
 jks.keypass=!keypass
 jks.storepass=!storepass
 jks.passlength=10
+jks.host=0.0.0.0
```

Comment:
The certificate must still be labeled for "localhost" or else the software will break.  This value cannot change.

This has less to do with signing and more to do with how the websocket's scope is advertised to the pc.

Let's add a new parameter for this called `wss.scope=0.0.0.0`